test/expected/build/library.j/dest/resources/library/j/library-core.js

```diff
@@ -0,0 +1,34 @@
+/*!
```

Comment:
In order to have the correct library namespace reflected, this file should be named `library/j/core/library.js`.

Reply:
Done